contrib/experimental/great_expectations_experimental/expectations/expect_multicolumn_values_to_be_between.py

```diff
@@ -0,0 +1,199 @@
+"""
+import functools
+import operator
+from typing import Optional
+
+from great_expectations.core.expectation_configuration import \
+    ExpectationConfiguration
+from great_expectations.exceptions import InvalidExpectationConfigurationError
+from great_expectations.execution_engine import (PandasExecutionEngine,
+                                                 SparkDFExecutionEngine,
+                                                 SqlAlchemyExecutionEngine)
+from great_expectations.expectations.expectation import \
+    MulticolumnMapExpectation
+from great_expectations.expectations.metrics.import_manager import F
+from great_expectations.expectations.metrics.map_metric_provider import (
+    MulticolumnMapMetricProvider, multicolumn_condition_partial)
+
+
+# This class defines a Metric to support your Expectation.
+# For most MulticolumnMapExpectations, the main business logic for calculation will live in this class.
+class ExpectMulticolumnValuesToBeBetween(MulticolumnMapMetricProvider):
+    # This is the id string that will be used to reference your metric.
+    condition_metric_name = "multicolumn_values.value_between"
+    # These point your metric at the provided keys to facilitate calculation
+    condition_domain_keys = (
+        "batch_id",
+        "table",
+        "column_list",
+        "row_condition",
+        "condition_parser",
+        "ignore_row_if",
+    )
+    condition_value_keys = (
+        "min_value",
+        "max_value",
+    )
+
+    # This method implements the core logic for the PandasExecutionEngine
+    @multicolumn_condition_partial(engine=PandasExecutionEngine)
+    def _pandas(cls, column_list, min_value, max_value, **kwargs):
+        columns_to_sum = column_list.columns
+        sum_of_columns = column_list.sum(axis=1)
+        return (sum_of_columns >= min_value) & (sum_of_columns <= max_value)
+
+    # This method defines the business logic for evaluating your metric when using a SqlAlchemyExecutionEngine
+    # @multicolumn_condition_partial(engine=SqlAlchemyExecutionEngine)
+    # def _sqlalchemy(cls, column_list, **kwargs):
+    #     raise NotImplementedError
+
+    # This method defines the business logic for evaluating your metric when using a SparkDFExecutionEngine
+    @multicolumn_condition_partial(engine=SparkDFExecutionEngine)
+    def _spark(cls, column_list, min_value, max_value, **kwargs):
+        columns_to_sum = column_list.columns
+        return (
+            functools.reduce(operator.add, [F.col(column) for column in columns_to_sum])
+            >= F.lit(min_value)
+        ) & (
+            functools.reduce(operator.add, [F.col(column) for column in columns_to_sum])
+            <= F.lit(max_value)
+        )
+
+
+# This class defines the Expectation itself
+class ExpectMulticolumnValuesToBeBetween(MulticolumnMapExpectation):
+    """Expect a row of multiple column to be in between given value"""
```

Comment:
Would you mind adding more detail to this docstring? Particularly to clarify that the expectation is based on the sum of values. Also I think you might consider renaming the expectation to mention that it is on the sum of values over the columns.
Lastly, could you add the expectation arguments?